app/containers/Root.js

```diff
@@ -81,18 +83,18 @@ class Root extends React.Component {
                   exact
                   path="/onboarding"
                   render={() => (
-                    <Modal withClose={hasWallets} onClose={() => history.push('/home')}>
+                    <ModalOverlay withClose={hasWallets} onClose={() => history.push('/home')}>
                       <Onboarding />
-                    </Modal>
+                    </ModalOverlay>
                   )}
                 />
                 <Route
                   exact
                   path="/syncing"
                   render={() => (
-                    <Modal withHeader onClose={() => history.push('/logout')} pb={0} px={0}>
+                    <ModalOverlay withHeader onClose={() => history.push('/logout')} pb={0} px={0}>
```

Comment:
Is it necessary to use `pb={0} px={0}` ?  I suppose we are overriding some defaults here ?

Reply:
Exactly, the modal has sensible defaults. in this case though, we want the content to stretch right the to bottom and sides.